examples/sssp/sssp.cu

```diff
@@ -34,9 +33,6 @@ void test_sssp(int num_arguments, char** argument_array) {
   
   auto [G, meta] = graph::build::from_csr_t<memory_space_t::device>(&csr);
```

Comment:
I'll think about it some more, we really need a way to get rid of the meta/host-graph entirely. And just be able to call G. I need to experiment maybe with a few more versions that may allow that.

Reply:
Maybe have `G` be on the host w/ a pointer `G->device` to the device graph? 

In `python_essentials`, I have a wrapper around `G` and `meta`: 
https://github.com/cfld/python_essentials/blob/master/gunrock_sssp.cu#L13

That sortof accomplishes something similar but I don't think is really ideal.

Reply:
Yeah, that's what I was thinking too, but also, I like the `device` and `host` a but because you know exactly which data you're accessing. I'll write a version that does that soon.